package.json

```
@@ -106,6 +107,8 @@
     "test:user:watch": "karma start --no-single-run --auto-watch --browsers ChromeHeadlessNoSandbox test/integration/user/karma.conf.js",
     "test:user:watchc": "karma start --no-single-run --browsers Chrome test/integration/user/karma.conf.js",
     "test:browser": "karma start --no-single-run --browsers Chrome test/unit/karma.conf.js",
+    "test:browsers:connect": "sc -u $SAUCELABS_USER -k $SAUCELABS_KEY -i cartovl-tunnel",
```

In my case, sauce labs is detecting a proxy url automatically and therefore it fails. By adding this flag it gets fixed.

```suggestion
"test:browsers:connect": "sc -u $SAUCELABS_USER -k $SAUCELABS_KEY -i cartovl-tunnel --no-autodetect"
```